vm/src/stdlib/os.rs

```diff
@@ -1191,6 +1191,27 @@ fn os_chmod(
     Ok(())
 }
 
+#[cfg(unix)]
+fn os_execv(
+    path: PyStringRef,
+    argv_list: PyIterable<PyStringRef>,
+    vm: &VirtualMachine,
+) -> PyResult<()> {
+    let path = ffi::CString::new(path.as_str()).unwrap();
+
+    let argv_list: Vec<ffi::CString> = argv_list
+        .iter(vm)
+        .unwrap()
```

Comment:
what's happened if `iter()` returns `Err`? `unwrap` will cause crash. We must relay out the exception. `.iter(vm)?` will do it.

Reply:
Thx for the review. Expect a fix for this issue in the upcoming commit, which I’ll push asap after correct exception handling has been implemented